### PR TITLE
SPCTR-2136 Improvement: Control - Each control should have help text support and verify the help text CSS consistency.

### PR DESCRIPTION
**Pull request by AbhijitNage123**
### Description

- https://brainstormforce.atlassian.net/browse/SPCTR-2136

### Screenshots

**Help/note CSS updated to same** - https://share.getcloudapp.com/xQuO5doO
- Image Dimention - https://share.getcloudapp.com/xQuOkNeA
- Multi Select Control - https://share.getcloudapp.com/OAu2d7z0
- Typography - https://share.getcloudapp.com/KounRNqn
- Responsive Slider - https://share.getcloudapp.com/QwujyBlD
- Text Shadow - https://share.getcloudapp.com/5zu2RbXb
- Background - https://share.getcloudapp.com/rRu5dmo7
- Tab control - https://share.getcloudapp.com/kpuOzRNk
- Border - https://share.getcloudapp.com/bLuONyqX
- Presets - https://share.getcloudapp.com/xQuOkNzb

### Types of changes

- Improvement in existing control

### How has this been tested?

- Added to the card.

### Checklist:
- [x] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [x] I've included any necessary tests 
- [ ] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: src/components/image/index.js [hunk 1]
```diff
@@ -258,7 +258,7 @@ const UAGMediaPicker = ( props ) => {
 								) }
 							</div>
 							{ props.help && (
```
Comment (rahulvarma722) [inline src/components/image/index.js:260]: please add help in spread operator

File: src/components/matrix-alignment-control/index.js [hunk 2]
```diff
@@ -61,6 +62,10 @@ const SpectraMatrixControl = ( props ) => {
 				onChange={ ( onChange || setAttributes ) ? ( newValue ) => onChangeHandler( newValue ) : false }
 			/>
 		</div>
+		{ props.help && (
```
Comment (rahulvarma722) [inline src/components/matrix-alignment-control/index.js:65]: add help in spread operator

File: src/components/multi-buttons-control/index.js [hunk 3]
```diff
@@ -181,7 +181,7 @@ const MultiButtonsControl = ( props ) => {
 				</div>
 				{ output[ deviceType ] ? output[ deviceType ] : output.Desktop }
 				{ props.help && (
```
Comment (rahulvarma722) [inline src/components/multi-buttons-control/index.js:183]: add help in spread operator

File: src/components/multi-buttons-control/index.js [hunk 4]
```diff
@@ -247,7 +247,7 @@ const MultiButtonsControl = ( props ) => {
 					) ) }
 				</ButtonGroup>
 				{ props.help && (
-					<p className="uag-control-help-notice">{ props.help }</p>
```
Comment (rahulvarma722) [inline src/components/multi-buttons-control/index.js:250]: add help in spread operator

File: src/components/multimedia-select/index.js [hunk 5]
```diff
@@ -91,7 +91,7 @@ const MultiMediaSelector = ( props ) => {
 				) }
 			</div>
 			{ props.help && (
-				<p className="uag-control-help-notice">{ props.help }</p>
+				<p className="components-base-control__help">{ props.help }</p>
```
Comment (rahulvarma722) [inline src/components/multimedia-select/index.js:94]: add help in spread operator

File: src/components/number-control/index.js [hunk 6]
```diff
@@ -194,7 +194,7 @@ const UAGNumberControl = ( props ) => {
 					/>
 				</div>
 				{ props.help && (
-					<p className="uag-control-help-notice">{ props.help }</p>
```
Comment (rahulvarma722) [inline src/components/number-control/index.js:197]: add help in spread operator

File: src/components/multimedia-select/index.js (hunk 5 above)
Comment (AbhijitNage123) [reply src/components/multimedia-select/index.js]: Done

File: src/components/multi-buttons-control/index.js (hunk 4 above)
Comment (AbhijitNage123) [reply src/components/multi-buttons-control/index.js:250]: Done

File: src/components/multi-buttons-control/index.js (hunk 3 above)
Comment (AbhijitNage123) [reply src/components/multi-buttons-control/index.js]: Done

File: src/components/matrix-alignment-control/index.js (hunk 2 above)
Comment (AbhijitNage123) [reply src/components/matrix-alignment-control/index.js]: Done

File: src/components/image/index.js (hunk 1 above)
Comment (AbhijitNage123) [reply src/components/image/index.js]: Done

File: src/components/icon-picker/index.js [hunk 7]
```diff
@@ -74,6 +74,9 @@ const UAGIconPicker = ( props ) => {
 					} }
 				/>
 			) }
+			{ props.help && (
```
[the remark this answers is not in the document]
Comment (AbhijitNage123) [reply src/components/icon-picker/index.js]: Done

**Review by RajkiranBagal — APPROVED** [review]

**Review by rahulvarma722 — APPROVED** [review]

**Pull request merged**